docs/custom_installs/local_rtd_vm.rst

```diff
@@ -1,20 +1,26 @@
 Local VM Install
 ================
+Please also consult instructions here which are updated more frequently:
+https://docs.readthedocs.io/en/stable/development/install.html
 
 Assumptions and Prerequisites
 -----------------------------
 
-* Debian VM provisioned with python 2.7.x
-* All python dependencies and setup tools are installed:
+* Debian VM provisioned with python 3.6.x
+* All dependencies are installed:
 
 .. prompt:: bash $
 
-  sudo apt-get install python-setuptools
   sudo apt-get install build-essential
-  sudo apt-get install python-dev
-  sudo apt-get install libevent-dev
+  sudo apt-get install python-pip python-setuptools
```

Comment:
I'm not 100% sure, but aren't those named like `python3-pip` and `python3-setuptools`?

Reply:
Probably `python3.6-pip and python3.6-setuptools`?